chapter_1/Functions/functions(1).py

```diff
@@ -0,0 +1,26 @@
+import socket
+
+
+def ret_banner(ip, port):
+    try:
+        socket.setdefaulttimeout(2)
+        s = socket.socket()
+        s.connect((ip, port))
+        banner = s.recv(1024)
+        return banner
+    except:
+        return
+
+
+def main():
+    ip1 = '192.168.95.148'
+    ip2 = '192.168.95.149'
+    port = '21'
```

Comment:
Also, this variable has the wrong data type. Do you see what I mean? Please check the book.

Reply:
Done.